### ISSUE TITLE
Memory error when copying large file

### ISSUE THREAD

**Issue by morphje** [ansible/ansible#18418]
##### ANSIBLE VERSION
ansible 2.2.0.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides

##### CONFIGURATION
No changes

##### OS / ENVIRONMENT
Ubuntu 16.04 with ansible/ansible PPA

##### SUMMARY
When copying a large file from local to a remote host, ansible crashed. Looks like a regression from fix #16392. Proposed fix worked in 2.1, however when updating to 2.2 the bug resurfaced despite having the fix merged.

##### STEPS TO REPRODUCE
```
- name: Copy large file
  win_copy:
    src: largeinstallerfile2GB.zip
    dest: 'c:\install'
```

##### ACTUAL RESULTS
TASK [windows : Copy largeinstallerfile2GB.zip] ***
task path: /root/ansible-test/roles/windows/tasks/install.yml:6
An exception occurred during task execution. The full traceback is:
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/ansible/executor/task_executor.py", line 119, in run
    res = self._execute()
  File "/usr/lib/python2.7/dist-packages/ansible/executor/task_executor.py", line 490, in _execute
    result = self._handler.run(task_vars=variables)
  File "/usr/lib/python2.7/dist-packages/ansible/plugins/action/copy.py", line 153, in run
    source_full = self._loader.get_real_file(source_full)
  File "/usr/lib/python2.7/dist-packages/ansible/parsing/dataloader.py", line 402, in get_real_file
    if is_encrypted_file(f):
  File "/usr/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py", line 158, in is_encrypted_file
    b_vaulttext = to_bytes(to_text(vaulttext, encoding='ascii', errors='strict'), encoding='ascii', errors='strict')
  File "/usr/lib/python2.7/dist-packages/ansible/module_utils/_text.py", line 177, in to_text
    return obj.decode(encoding, errors)
MemoryError

fatal: [ago-win1]: FAILED! => {
    "failed": true,
    "msg": "Unexpected failure during module execution.",
    "stdout": ""
}

**Comment by alikins** [comment]
I believe ed134d81f1b66fed63aa49b3599dd5b8388ff4b4 should fix this (and de7fbd407a787c538b5d7360fbb69a2c68bca380 in the stable-2.2 branch for the next 2.2 release)

**Comment by bcoca** [comment]
duplicate of #16391
